packages/mdc-icon-button/_mixins.scss

```diff
@@ -113,22 +125,41 @@
   $feat-color: mdc-feature-create-target($query, color);
   $feat-structure: mdc-feature-create-target($query, structure);
 
-  @include mdc-icon-button-size($mdc-icon-button-size, $query: $query);
-
   @include mdc-feature-targets($feat-structure) {
-    display: inline-block;
+    display: flex;
     position: relative;
+    align-items: center;
+    justify-content: center;
     box-sizing: border-box;
     border: none;
     outline: none;
     background-color: transparent;
     fill: currentColor;
     color: inherit;
+    font-size: $mdc-icon-button-icon-size;
     text-decoration: none;
     cursor: pointer;
     user-select: none;
   }
 
+  // For customizing icon-button display, we need to increase specifity to ensure
+  // overrides apply. Styles defined in the .material-icons CSS class are
+  // loaded separately, so the order of CSS definitions is not guaranteed.
+  &:not(:disabled) {
+    @include mdc-feature-targets($feat-structure) {
+      display: flex;
```

Comment:
If you're adding this here you can remove display CSS property from above.

Reply:
Done.